qiskit/circuit/quantumcircuit.py

```diff
@@ -355,6 +356,23 @@ def from_instructions(
             circuit._append(instruction)
         return circuit
 
+    @property
+    def layout(self) -> Optional[TranspileLayout]:
+        r"""Return any associated layout information anout the circuit
+
+        This attribute contains an optional :class:`~.TranspileLayout`
+        object. This is typical set on the output from :func:`~.transpile`
```

Comment:
```suggestion
        object. This is typically set on the output from :func:`~.transpile`
```

Reply:
Fixed in: https://github.com/Qiskit/qiskit-terra/pull/9486/commits/fa93ec328094c796af05aafec825480356a2e5a8